src/frameworks/assemble.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "assemble",
+  "category": "static_site_generator",
+  "detect": {
+    "npmDependencies": ["assemble"],
+    "excludedNpmDependencies": [],
+    "configFiles": []
+  },
+  "watch": {},
```

Comment:
Seems like the project is not active anymore:
https://github.com/assemble/assemble/
Couldn't get a local example working.

Reply:
Well, at least we won't break this! :D